mt-kahypar/partition/context_enum_classes.cpp

```diff
@@ -266,10 +289,10 @@ namespace mt_kahypar {
   InitialPartitioningMode initialPartitioningModeFromString(const std::string& mode) {
     if (mode == "direct") {
       return InitialPartitioningMode::direct;
-    } else if (mode == "recursive") {
-      return InitialPartitioningMode::recursive;
-    } else if (mode == "recursive_bisection") {
-      return InitialPartitioningMode::recursive_bisection;
+    } else if (mode == "deep_multilevel") {
```

Comment:
Yeah, we could use the same enum and forbid `direct_kway` for the initial partitioning mode. Note, this will change the naming of the parameter for initial partitioning

Reply:
Ah, actually we would probably rather use one value for both direct-k-way partitioing and direct initial partitioning mode